index.js

```diff
@@ -40,11 +40,41 @@ To save you from having to count the items above, you can assume that length of
 
 i.e. is31Flavors(originalFlavors) will return TRUE.*/
 
-function is31Flavors(/*code here*/){
-
-    /*code here*/
-
+function is31Flavors(arr, string){
+    var originalFlavors = ["Banana Nut Fudge",
```

Comment:
Since the array above is global, you can access it within this function without having to define `originalFlavors` inside of this scope. The question is asking you to create a condition that returns True or False if the length of the array is 31.